app/services/bulk_update/placements/import_row.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+module BulkUpdate
+  module Placements
+    class ImportRow
+      include ServicePattern
+
+      def initialize(placement_row)
+        @placement_row = placement_row
+        @urn = placement_row.urn
+      end
+
+      def call
+        return unless placement_row.can_be_imported?
+
+        placement_row.row_errors.destroy_all
```

Comment:
I'm wondering if we should wrap all these database calls in a transaction so we can roll everything back if something goes wrong. Could also do this as a follow up hardening ticket, not a blocker right now!